opacus/privacy_engine.py

```diff
@@ -185,7 +238,59 @@ def make_private(
         noise_seed: Optional[int] = None,
         poisson_sampling: bool = True,
         clipping: str = "flat",
-    ):
+    ) -> Tuple[GradSampleModule, DPOptimizer, DataLoader]:
+        """
+        Add privacy-related responsibilites to the main PyTorch training objects:
+        model, optimizer and the data loader.
+
+        All of the returned objects act just like treir non-private counterparts
+        passed as arguments, but with added DP tasks.
+
+        Model is wrapped to also compute per sample gradients.
+        Optimizer is now responsible for gradient clipping and adding noise to the
+            gradients
+        DataLoader is updated to perform Poisson sampling.
+
+        Note, that using any other models, optimizers or data sources during training
+        will invalidate stated privacy guarantees.
+
+        Args:
+            module: PyTorch module to be used for training
+            optimizer: Optimizer to be used for training
+            data_loader: DataLoader to be used for training
+            noise_multiplier: The ratio of the standard deviation of the Gaussian noise to
+                the L2-sensitivity of the function to which the noise is added
+                (How much noise to add)
+            max_grad_norm: The maximum norm of the per-sample gradients. Any gradient with norm
+                higher than this will be clipped to this value.
+            batch_first: Flag to indicate if the input tensor to the corresponding module
+                has the first dimension representing the batch. If set to True, dimensions on
+                input tensor will be ``[batch_size, ..., ...]``.
+            loss_reduction: Indicates if the loss reduction (for aggregating the gradients)
+                is a sum or a mean operation. Can take values "sum" or "mean"
+            noise_seed: Seed to be used for random noise generation
+            poisson_sampling: ``True`` if you want to use standard sampling required
+                for DP guarantees. Setting ``False`` will leave provided data_loader
+                unchanged. Technically this doesn't fit the assumptions made by
+                privacy accounting mechanism, but it can be a good approximation when
+                using Poisson sampling is unfeasible.
+            clipping: Per sample gradient clipping mechanism ("flat" or "per_layer").
+                Flat clipping calculates the norm of the entire gradient over
+                all parameters, while per layer clipping sets individual norms for
+                every parameter tensor. Flat clipping is usually preferred, but using
+                per layer clipping in combination with distributed training can provide
+                notable performance gains.
+
+        Returns:
+            Tuple of (model, optimizer, data_loader)
```

Comment:
This in the website will be something like this: "Tuple of (model, optimizer, data_loader)Model is a wrapper...". Perhaps can put a dot and space at the end of this line